strapi/strapi
# feat: extend i18n attributes to every content type

### What does it do?

Extend every content type to have `locale` and `localizations` (the later will be completely removed during the v5 development) attributes.

### Why is it needed?

We will allow disabling i18n from a content type, but the locale attribute will always be present in the database. If i18n is disabled for the CT, the locale value will be `null`.


### How to test it?
See the response value of a content type with disabled i18n, it should have the locale attribute set to null. The i18n enabled content types should still keep a non null locale value.
